tests/robot-tests/tests/admin_and_public/bau/archive_publication.robot

```diff
@@ -0,0 +1,346 @@
+${RELEASE_NAME_SUPERSEDE}=          Financial Year 2000-01
+${SUBJECT_NAME_SUPERSEDE}=          Subject for superseding publication
+
+*** Test Cases ***
+Create new publication to be archived and release via API
+    ${PUB_ID_ARCHIVE}=    user creates test publication via api    ${PUBLICATION_NAME_ARCHIVE}
+    user create test release via api    ${PUB_ID_ARCHIVE}    FY    3000
+
+Navigate to archive-publication release
+    user navigates to editable release summary from admin dashboard    ${PUBLICATION_NAME_ARCHIVE}
+    ...    ${RELEASE_NAME_ARCHIVE} (not Live)
+
+Import archive-publication subject to release
+    user uploads subject    ${SUBJECT_NAME_ARCHIVE}    upload-file-test.csv    upload-file-test.meta.csv
+
+Add data guidance to archive-publication subject
+    user clicks link    Data guidance
+    user waits until h2 is visible    Public data guidance    %{WAIT_SMALL}
+
+    user waits until page contains element    id:dataGuidance-dataFiles
+    user waits until page contains accordion section    ${SUBJECT_NAME_ARCHIVE}
+    user enters text into data guidance data file content editor    ${SUBJECT_NAME_ARCHIVE}
+    ...    ${SUBJECT_NAME_ARCHIVE} data guidance content
+
+    user clicks button    Save guidance
+    user waits until page contains button    Edit guidance
+
+Go to "Sign off" page and approve archive-publication release
+    user clicks link    Sign off
+    user approves original release for immediate publication
+
+Create new publication to supersede other publication and release via API
+    ${PUB_ID_SUPERSEDE}=    user creates test publication via api    ${PUBLICATION_NAME_SUPERSEDE}
+    user create test release via api    ${PUB_ID_SUPERSEDE}    FY    2000
+
+Set archive-publication to be superseded by superseding-publication
+    user navigates to admin dashboard
+    user opens accordion section    ${PUBLICATION_NAME_ARCHIVE}
+    ${ARCHIVE_ACCORDION}=    user gets accordion section content element    ${PUBLICATION_NAME_ARCHIVE}
+    user clicks link    Manage publication    ${ARCHIVE_ACCORDION}
+
+    user waits until h1 is visible    Manage publication    %{WAIT_SMALL}
+    user waits until page contains element    id:publicationForm-supersede
+
+    user chooses select option    id:publicationForm-supersededById    ${PUBLICATION_NAME_SUPERSEDE}
+
+    user clicks button    Save publication
+    user waits until modal is visible    Confirm publication changes
+    user clicks button    Confirm
+    user waits until modal is not visible    Confirm publication changes
+
+Validate archive warning is on Admin dashboard for archive-publication release
+    ${accordion}=    user opens publication on the admin dashboard    ${PUBLICATION_NAME_ARCHIVE}
+    user checks element should contain    ${accordion}
+    ...    This publication will be archived when its superseding publication has a live release published.
+
+Check public site, that archive-publication still appears correctly
```

Comment:
Would word this a little better. Something such as Validate that archive-publication appears correctly on public frontend`